dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -128,6 +128,16 @@ def dataset_compatible_tensorflow():
   from tensorflow.python.data.util import structure
   import functools
 
+  def dataset_options():
+    options = tf.data.Options()
+    try:
+        options.experimental_optimization.apply_default_optimizations = False
+        options.experimental_optimization.autotune = False
+    except:
+        print('Could not set TF Dataset Options')
```

Comment:
1. Why are we ignoring this error?
2. If we print anything, how about printing the message from the underlying exception?

Reply:
This was leftover from when I was developing this. Basically, older TF versions did not have the second option to set, but it was fine, because the behavior of the whole thing was as we wanted it to be. I removed `try`, because all TF versions that we support now do have this.